src/routes/profile-route/profile-sidebar.jsx

```diff
@@ -8,7 +8,7 @@ import sidebarHeart from "../../images/sidebar-heart.svg";
 import sidebarWrench from "../../images/sidebar-wrench.svg";
 
 export const ProfileSidebar = () => {
-  const { setIsLoggedIn } = useCurrentUser();
+  const { setIsLoggedIn, setCurrentUser } = useCurrentUser();
```

Comment:
We're not using the new variable we're bringing in, let's remove this.

Reply:
Ok, I've updated the branch!